leetcode/69.c

```diff
@@ -0,0 +1,31 @@
+//method-1
+
+
+
+//According to the problem statement we must not use any "built-in exponent function" so by considering that...
+
+int mySqrt(int x){
+
+//a new variable which runs till we reach the exact number or the number which is just exactly greater than the number(x).
+  long long int k=1;
+  while(k*k <= x ){
+     k++;
+  }
+  // 0 is also a non-negative integer so considering that we can use.
+  if(x==0)  return 0;
```

Comment:
Please follow the same style in all code. So you dont use brackets there, and use it in the `else`. Please use brackets there also.